modules/rosa-create-objects.adoc

```diff
@@ -117,7 +117,13 @@ $ rosa create cluster --cluster-name=<cluster_name> [arguments]
 |Option |Definition
 
 |--additional-compute-security-group-ids <sec_group_id>
-|The identifier of one or more additional security groups to use in addition to the default security groups. For more information on additional security groups, see the requirements for _Security groups_ under _Additional resources_.
+|The identifier of one or more additional security groups to use in addition to the default security groups to use with the standard machine pool created along side the cluster. For more information on additional security groups, see the requirements for _Security groups_ under _Additional resources_.
```

Comment:
```suggestion
|The identifier of one or more additional security groups to use in addition to the default security groups, which are used with the standard machine pool created along side the cluster. For more information on additional security groups, see the requirements for _Security groups_ under _Additional resources_.
```

Reply:
I think "that" is reads better here, instead of another "to use". This is a long sentence, and the repeating "to use" makes it hard to connect the parts.

Reply:
This is a global comment, since I see this used other places

Reply:
changed where necessary